test/io-tests/configs/no-defaults.env

```diff
@@ -0,0 +1,26 @@
+PGRST_DB_URI="tmp_db"
+PGRST_DB_SCHEMA="multi,   tenant,setup"
+PGRST_DB_ANON_ROLE="root"
+PGRST_DB_POOL=1
+PGRST_DB_POOL_TIMEOUT=100
+PGRST_DB_EXTRA_SEARCH_PATH="public, test"
+PGRST_DB_CHANNEL="postgrest"
+PGRST_DB_CHANNEL_ENABLED=true
+PGRST_DB_TX_END="rollback-allow-override"
+PGRST_DB_PREPARED_STATEMENTS=false
+PGRST_SERVER_HOST="0.0.0.0"
+PGRST_SERVER_PORT=80
+PGRST_SERVER_UNIX_SOCKET="/tmp/pgrst_io_test.sock"
+PGRST_SERVER_UNIX_SOCKET_MODE="777"
+PGRST_OPENAPI_SERVER_PROXY_URI="https://postgrest.org"
+PGRST_JWT_SECRET="c2VjdXJpdHl0aHJvdWdob2JzY3VyaXR5"
+PGRST_JWT_AUD="https://postgrest.org"
+PGRST_SECRET_IS_BASE64=true
+PGRST_ROLE_CLAIM_KEY=".user[0].\"real-role\""
+PGRST_MAX_ROWS=1000
+PGRST_PRE_REQUEST="please_run_fast"
+PGRST_ROOT_SPEC="openapi_v3"
+PGRST_RAW_MEDIA_TYPES="application/vnd.pgrst.config"
+PGRST_LOG_LEVEL="info"
+PGRST_APP_SETTINGS_test="test"
+PGRST_APP_SETTINGS_test2="test"
```

Comment:
`PGRST_APP_SETTINGS_xxx` variables are currently read in as-is, without any transformation on the `xxx` part. I am unsure whether we should apply some transformations like making them lowercase and replacing `_` with `.` or something like that.

Reply:
I think it's good to leave them as-is. That way it corresponds to the config file option - which AFAICT can use lower/uppercase, same for `_` or `.`.